arch/arm/soc/st_stm32/stm32l0/gpio_registers.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ * Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _STM32L0X_GPIO_REGISTERS_H_
+#define _STM32L0X_GPIO_REGISTERS_H_
+
+/**
+ * @brief
+ *
+ * Based on reference manual:
+ *   STM32L0X advanced ARM ® -based 32-bit MCUs
+
+ *
+ * Chapter 8: General-purpose I/Os (GPIO)
+ * Chapter 9: System configuration controller (SYSCFG)
+ */
+
+struct stm32l0x_gpio {
+	u32_t moder;
+	u32_t otyper;
+	u32_t ospeedr;
+	u32_t pupdr;
+	u32_t idr;
+	u32_t odr;
+	u32_t bsrr;
+	u32_t lckr;
+	u32_t afr[2];
+	u32_t brr;
+};
+
+union syscfg_cfgr1 {
```

Comment:
I think we can just leave syscfg_cfgr1 as u32 at the moment.

Reply:
Let's remove this union completely. Maybe we'll add correct L0 version later, but it is unneeded ATM.